### PR TITLE
Fix order of css rules

### PR DESCRIPTION
**Pull request by nicodh**
since v1.77.7 the behaviour of compiling css from nested rules changed: https://sass-lang.com/documentation/breaking-changes/mixed-decls/

To avoid warnings and keep the old structure of the compiled css all properties should occur before nested rules

Follow up of #4054

### CODE REVIEW

**Comment by WofWca** [comment]
So, is this just a refactor? Have you seen if something is bugged without this, but with the upgrade of SASS?

File: src/renderer/components/screens/WelcomeScreen/OnboardingScreen.tsx [hunk 1]
```diff
@@ -38,8 +38,12 @@ export default function OnboardingScreen(props: Props) {
...
       )
       if (acInfo.kind === 'Unconfigured') {
-        await props.onUnSelectAccount()
-        await EffectfulBackendActions.removeAccount(props.selectedAccountId)
+        try {
+          await props.onUnSelectAccount()
+          EffectfulBackendActions.removeAccount(props.selectedAccountId)
+        } catch (error) {
+          log.error('error on removeAccount', error)
+        }
```
Comment (WofWca) [inline src/renderer/components/screens/WelcomeScreen/OnboardingScreen.tsx:46]: Unrelated?

Comment (nicodh) [reply src/renderer/components/screens/WelcomeScreen/OnboardingScreen.tsx:46]: Oops - yes. Will remove it

**Comment by nicodh** [comment]
There were warnings when building. In future the generated css will look different then before. With these changes the generated css should stay the same, at least according to the documentation

**Review by WofWca — APPROVED** [review]
Except for the unrelated change, looks good.

**Review by WofWca — APPROVED** [review]
